src/components/NoteCard/NoteCard.jsx

```diff
@@ -48,20 +54,51 @@ function NoteCard({
 
       <small>{note}</small>
 
-      <div className="d-flex note-footer mt-2">
+      <div className="d-flex mt-1 flex-wrap flex-gap-small">
+        {tags.map((tag) => {
+          return <small className="note-label-priority"> {tag} </small>;
+        })}
+      </div>
+
+      <div className="d-flex note-footer mt-1">
         <div className="d-flex note-footer note-label-priority-container">
-          <small className="note-label-priority">Health</small>
           <small className="note-label-priority"> {priority} </small>
         </div>
-        <div className="d-flex note-footer note-icons-container">
-          <i
-            className="fa-solid fa-pencil"
-            onClick={() => handleUpdateNote(_id)}
-          ></i>
-          <i className="fa-solid fa-box-archive"></i>
+        <div
+          className={`d-flex note-footer ${
+            location.pathname == "/archives"
+              ? "archivenote-icons-container"
+              : "note-icons-container"
+          } `}
+        >
+          {location.pathname !== "/archives" && (
+            <i
+              className="fa-solid fa-pencil"
+              onClick={() => handleUpdateNote(_id)}
+            ></i>
+          )}
+
+          {location.pathname !== "/archives" && (
```

Comment:
Instead of checking condition three times, you can check only one time na,

You can do something like,
```jsx
{location.pathname !== "/archives" && (
       <>
              <i></i>
              <i></i>
              ... others
      </>
    )}
```